packages/ui/src/components/Alert/alert.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Alert as DefaultAlert, AlertProps as DefaultAlertProps } from 'antd'
+import classNames from 'classnames'
+
+type CondoAlertProps = {
+    type: 'success' | 'info' | 'warning' | 'error';
+}
+
+const ALERT_CLASS_PREFIX = 'condo-alert'
+
+export type AlertProps = Omit<DefaultAlertProps, 'type' | 'prefix' | 'prefixCls' | 'banner' | 'afterClose' | 'closable' | 'closeText' | 'closeIcon' | 'onClose'> & CondoAlertProps
```

Comment:
How about showIcon and icon?

Reply:
you should also omit style attribute

Reply:
I guess it's easier to Pick needed props here rather the omit rest

Reply:
Yes, we probably want to expose only the required for us and supported by our team component API